server/data/mappers/inPrisonCourseMapper.ts

```diff
@@ -1,17 +1,17 @@
 import moment from 'moment'
 import type { LearnerEducation } from 'curiousApiClient'
-import type { InPrisonEducation } from 'viewModels'
+import type { InPrisonCourse } from 'viewModels'
 
-const toInPrisonEducation = (apiLearnerEducation: LearnerEducation): InPrisonEducation => {
+const toInPrisonCourse = (apiLearnerEducation: LearnerEducation): InPrisonCourse => {
   return {
     prisonId: apiLearnerEducation.establishmentId,
     prisonName: apiLearnerEducation.establishmentName,
     courseCode: apiLearnerEducation.courseCode,
     courseName: apiLearnerEducation.courseName,
-    courseStartDate: moment(apiLearnerEducation.learningStartDate).toDate(),
+    courseStartDate: moment(apiLearnerEducation.learningStartDate).utc().toDate(),
```

Comment:
Ensure the dates are UTC (one of the tests burnt me with this, hence the change 😁 )